src/FirstSteps/RPG/Items/OldChest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FirstSteps.RPG.Items
+{
+    class OldChest
+    {
+        private static List<Item> itemsForHuman = new List<Item>()
+        {
+            new Pitchfork(),
+            new Knife()
+        };
+        public static Item Open()
+        {
+            var random = new Random().Next(0,5);
+            if(random > itemsForHuman.Count())
```

Comment:
The method `itemsForHuman.Count()` returns the exact number of items on the list so in your case it returns 2.  So the condition `random > itemsForHuman.Count()` is false for 0, 1, and 2. But when random is equal to 2 you will get an error because you have only two items and  `itemsForHuman[2]` is out of range.